templates/app-autoscaler-deployment.yml

```diff
@@ -72,6 +72,10 @@ instance_groups:
         - name: postgres
           password: "((database_password))"
           tag: default
+        connection_config: &databaseConnectionConfig
```

Comment:
you need to have db connection settings in  external db as well.

Reply:
@boyang9527  by a bosh var-file?